lib/improver/metadata/amend.py

```diff
@@ -41,27 +41,27 @@
 
 
 def update_stage_v110_metadata(cube):
-    """Translates meta-data relating to the grid_id attribute from StaGE
+    """Translates metadata relating to the grid_id attribute from StaGE
     version 1.1.0 to later StaGE versions.
     Cubes that have no "grid_id" attribute are not recognised as v1.1.0 and
     are ignored.
 
     Args:
         cube (iris.cube.Cube):
-            Cube to modify meta-data in (modified in place)
+            Cube to modify metadata in (modified in place)
 
     Returns:
-        bool:
-            True if meta-data have been changed by this function.
+        cube (iris.cube.Cube):
+            The cube with updated metadata.
     """
     try:
         grid_id = cube.attributes.pop('grid_id')
     except KeyError:
-        # Not a version 1.1.0 grid, so exit.
-        return False
+        # Not a version 1.1.0 grid, do nothing
+        return cube
     cube.attributes.update(MOSG_GRID_DEFINITION[GRID_ID_LOOKUP[grid_id]])
     cube.attributes['mosg__grid_version'] = '1.1.0'
-    return True
+    return cube
```

Comment:
I'm not a great fan of modifying in place and returning from the same function.  Why not have no return statement and have a CLI that returns the input cube (which may or may not have been modified)?

Reply:
Discussed with @cgsandford, function now returns `None` (same behaviour as not `return`ing). Eventually this is going to be nuked anyway.